glide.yaml

```diff
@@ -41,3 +41,5 @@ import:
   - fv.containers
 - package: github.com/coreos/etcd
   version: v3.2.7
+- package: golang.org/x/crypto
+  version: 1351f936d976c60a0a48d728281922cf63eafb8d
```

Comment:
I think this version causes issues with calicoctl (especially /x/sys windows package) but should be fine in CNI. This pin doesn't seem relevant to this PR, was it conflicting with glide up?

Reply:
Oh I see your comment about --vendor now.